screwdriver.yaml

```diff
@@ -1,16 +1,15 @@
-workflow:
-    - publish
-
 shared:
-    image: node:6
+    image: node:8
 
 jobs:
     main:
+        requires: [~pr, ~commit]
         steps:
             - install: npm install
             - test: npm test
 
     publish:
+        requires: main
         steps:
             - install: npm install semantic-release
```

Comment:
we should use the semantic release template for this

Reply:
On it 👍